File: baldosas/solutions/partial/bkorecic-st1.cpp
#include <bits/stdc++.h>

using namespace std;
typedef long long ll;

// @ocimatic should-fail=[st2,st3,st4]
int main(){
  ll n, m;
  cin >> n >> m;

  ll white=0, black=0;
  for(ll i=0; i<n; i++){
    for(ll j=0; j<m; j++){
      if((i+j)%2==0) white++;
      else black++;
    }
  }
  cout << white << ' ' << black << '\n';
}

File: baldosas/solutions/partial/bkorecic-st1.py
# @ocimatic should-fail=[st2,st3,st4]

n, m = [int(x) for x in input().split()]

white = 0
black = 0
for i in range(n):
    for j in range(m):
        if (i+j)%2 == 0:
            white += 1
        else:
            black += 1
print(white, black)
